code/modules/mob/living/carbon/human/life.dm

```diff
@@ -892,9 +892,9 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 
 
 		//Vision //god knows why this is here
-		var/obj/item/organ/vision
+		var/obj/item/organ/internal/vision
 		if(species.vision_organ)
-			vision = internal_organs_by_name[species.vision_organ]
+			vision = get_int_organ(/obj/item/organ/internal/eyes)//wot?

```

Comment:
This won't look for the `vision_organ` at all - you'll want `get_int_organ(species.vision_organ)`, and you'll need to set vision_organ where appropriate to the correct classpath - for example, for IPCs, `/obj/item/organ/internal/optical_sensor`